src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -23,6 +23,7 @@ public IndexingConfiguration(RavenConfiguration root)
             _root = root;
         }
 
+        [DefaultValue(false)]
```

Comment:
Didn't we want also to validate that all of them have 'DefaultValue' ? This can be done in ConfigurationCategory.ValidateProperty

Reply:
done